pkg/operator/provisioner_test.go

```diff
@@ -20,18 +20,47 @@ import (
 	"fmt"
 	"testing"
 
+	"k8s.io/api/core/v1"
+
 	"strings"
 
+	"github.com/rook/rook/pkg/clusterd"
+	"github.com/rook/rook/pkg/operator/kit"
+	"github.com/rook/rook/pkg/operator/mon"
+	"github.com/rook/rook/pkg/operator/test"
 	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
 )
 
 func TestProcessMonAddresses(t *testing.T) {
-	unformattedMons := []string{"10.0.0.1:8124/0", "10.0.0.2:8125", "10.0.0.3/24"}
-	formattedMons := processMonAddresses(unformattedMons)
+	clientset := test.New(3)
+	context := &clusterd.Context{KubeContext: kit.KubeContext{MasterHost: "foo", Clientset: clientset}}
+	clusterMgr := newClusterManager(context, []inclusterInitiator{})
+	ns := "myns"
+	config := provisionerConfig{clusterName: ns}
+	p := &rookVolumeProvisioner{clusterManager: clusterMgr, provConfig: config}
+
+	// create the test set of mons
+	testConfigMap := &v1.ConfigMap{ObjectMeta: metav1.ObjectMeta{Name: mon.EndpointConfigMapName},
+		Data: map[string]string{mon.EndpointKey: "mon1=10.0.0.1:6790,mon2=10.0.0.2:8125,mon3=10.0.0.3:6790"}}
```

Comment:
i guess these values will always be formatted. So no subnet mask like 10.0.0.3/24?